topics/programming-language/index.md

```diff
@@ -0,0 +1,7 @@
+---
+display_name: Programming language
+short_description: A programming language is a language for developing apps and all other sorts of programs.
+
+topic: programming-language
+---
+A programming language is a language for developing apps, games all other sorts of programs. No programming language = plain text.
```

Comment:
This bit is a little confusing to me. Can it be reworded? I am not sure what "No programming language = plain text" means either.